config/routes.rb

```diff
@@ -1,3 +1,11 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :microposts
+  resources :users
+  
+  root 'static_pages#home'
+  get 'static_pages/home'
```

Comment:
This works? I've never seen a route declaration without a `to:` argument. Ah, the magic of Rails...